scripts/utils/releaseDocumentation.js

```diff
@@ -0,0 +1,53 @@
+const shellUtils = require('shell-utils');
+const exec = shellUtils.exec;
+const fs = require('fs');
+const includes = require('lodash/includes');
+
+const docsPath = `${process.cwd()}/website`;
+const docsVersionsJsonPath = `${docsPath}/versions.json`;
+
+function buildDocsForVersion(version) {
+  console.log(`Building documentation version: ${version}`);
+  const originalDir = process.cwd();
+
+  try {
+    process.chdir(docsPath);
+    exec.execSync(`npm install`);
+    exec.execSync(`npm run docusaurus docs:version ${version}`);
+    exec.execSync(`git add .`);
```

Comment:
What exactly finalizes the commit after adding the files to git? @asafkorem

Reply:
Nice catch @noomorph!
The docs version commit was part of the lerna publish command, but now that we publish the docs version after the npm version we should add a separated git commit.

Reply:
@asafkorem  but the problem is that you have to push to `gh-pages` also...

Reply:
@noomorph the deployment to GH pages is done separately by the "Website" GH action once the commit is pushed.